### DISCUSSION TITLE
Spatie Logging To Multiple Databases

### DISCUSSION THREAD

**Discussion by MasPav** [spatie/laravel-activitylog#859] [Q&A]
Hi Community, i would like to know if its possible to store your spatie logs in multiple databases ?

**Comment by Gummibeer** [comment]
It's not supported by default. It starts with Laravel not (easily) supporting one model across multiple database connections/tables.

So there are possibly hacky ways around by changing model properties on runtime but not tested or guaranteed ones. So on your own risk.
